tests/test_process.c

```diff
@@ -26,35 +26,35 @@
 #include <unistd.h>
 
 #include "sysrepo.h"
-#include "tests/config.h"
+#include "test_common.h"
 
-#define sr_assert(cond) if (!(cond)) { fprintf(stderr, "\"%s\" not true\n", #cond); sr_assert_line(); abort(); }
+#define sr_assert(cond) if (!(cond)) { TLOG_ERR("\"%s\" not true", #cond); sr_assert_line(); abort(); }
```

Comment:
Now all these prints will be affected by *sysrepo* log level, which is not ideal, but I can live with that, I guess.

Reply:
TLOG_ERR, TLOG_INF, TLOG_WRN will always print, regardless of the configured sysrepo log level.

Reply:
Sorry, missed that.